tests/NuGet.IndexingTests/ResponseFormatterTests.cs

```diff
@@ -65,16 +65,24 @@ public void WriteV2CountResultTest(int totalHits, string expected)
             Dictionary<string, string> commitUserData,
```

Comment:
https://github.com/NuGet/NuGetGallery/issues/3652 - Need to check this in first to add functional tests.